one.ablaze.floorp.yml

```diff
@@ -0,0 +1,39 @@
+app-id: one.ablaze.floorp
+name: Floorp
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+sdk: org.freedesktop.Sdk
+add-extensions:
+  org.fedoraproject.PlatformExtensions.ffmpeg:
+    directory: lib/ffmpeg
+    bundle: false
+    remove-after-build: false
+    subdirectories: true
+    
+command: floorp
+modules:                
+  - name: firefox
+    buildsystem: simple
+    build-commands:
+      - install -D floorp.tar.xz /app/floorp.tar.xz
+      - cd /app/ && tar Jxfv floorp.tar.xz && rm floorp.tar.xz
+    sources:
+      - type: file
+        url: 'https://sda1.net/storage/floorp/tarball/8.7.2/floorp.tar.xz'
+        sha256: ee9a2c31385c56ea43cd699d1ed9569147cdd56695e567b7a374fa4f2639b1c4
+
+finish-args:
+  - --share=ipc
+  - --socket=pcsc  
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=home
```

Comment:
it's a web browser, it should have a more restricted access to the filesystem.

Reply:
still

Reply:
Changed to allow access only to the configuration and download directories.